config/confighttp/confighttp.go

```diff
@@ -289,6 +289,14 @@ type ServerConfig struct {
 	CompressionAlgorithms []string `mapstructure:"compression_algorithms"`
 }
 
+type AuthConfig struct {
+	// Auth for this receiver.
+	*configauth.Authentication `mapstructure:"-"`
+
+	// RequestParameters is a list of parameters that should be extracted from the request and added to the context.
+	RequestParameters []string `mapstructure:"request_params"`
```

Comment:
I may be late to the discussion, but should this be part of `configauth.Authentication` instead? Would `configgrpc` need something similar?

Reply:
I think this is particular to HTTP: in gRPC, things are either metadata, or part of the message. The authenticator won't be able to (easily) get a parameter from the message without knowing the model used for that.